pkg/cmd/get.go

```diff
@@ -65,6 +76,8 @@ func NewGetCmd() *cobra.Command {
 
 	cmd.Flags().BoolVarP(&o.AllNamespaces, "all-namespaces", "A", o.AllNamespaces, "If present, list the requested object(s) across all namespaces. Namespace in current context is ignored even if specified with --namespace.")
 	cmd.Flags().StringVarP(&o.LabelSelector, "selector", "l", o.LabelSelector, "Selector (label query) to filter on, supports '=', '==', '!=', 'in', 'notin'.(e.g. -l key1=value1,key2=value2,key3 in (value3)). Matching objects must satisfy all of the specified label constraints.")
+	cmd.Flags().BoolVar(&o.InCluster, "in-cluster", true, "Include resources from the Kubernetes cluster.")
+	cmd.Flags().BoolVar(&o.Archived, "archived", true, "Include resources from KubeArchive.")
```

Comment:
I like [`BoolVarP`](https://pkg.go.dev/github.com/spf13/pflag#FlagSet.BoolVarP) over just `BoolVar` because it lets you specify a shorthand for the flag (for example: `--in-cluster` could also be `-c`)